.github/workflows/more-tests.yml

```diff
@@ -9,23 +9,17 @@ on:
 
 jobs:
   test-cuda:
-    uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
+    uses: pytorch/test-infra/.github/workflows/linux_job_v2.yml@main
```

Comment:
What's the difference between these two?

Reply:
Former is deprecated: https://github.com/pytorch/pytorch/issues/123649